File: solution/NoteManagementSolution/NoteService/Models/Note.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace NoteService.Models
{
    public class Note
    {
        public Note()
        {

        }

        public int ID { get; set; }
        public string NoteMessage { get; set; }
        public EnumNoteType NoteType { get; set; }
        public int UserID { get; set; }
        public DateTime NoteCreateTime { get; set; }
        public DateTime ReminderTime { get; set; }
        public string ErrorMessage { get; set; }
      }

    public enum EnumNoteType
    {
        Regular = 1,
        Reminder = 2,
        Task = 3,
        BookMark = 4
    }
}

File: NoteMgt/src/app/Models/Note.ts
export class Note {
    public id: number;
    public noteMessage: string;
    public noteType: number;
    public userID: number;
    public noteCreateTime: Date;
    public errorMessage: string;
    public reminderTime: Date;
    public constructor() {
        this.id = 0;
        this.noteMessage = "";
        this.noteType = 0;
        this.userID = 0;
        this.noteCreateTime = new Date();
        this.reminderTime = new Date();
        this.errorMessage = "";
    }
}